src/Extension/Tasks/AcceptanceTestingTask.yaml

```diff
@@ -0,0 +1,12 @@
+---
+id: 'tests:sdk:acceptance'
+short_description: 'Run acceptance tests'
+help: ~
+stage: build
+version: 0.1.0
+command: 'php %sdk_dir%/vendor/bin/codecept run acceptance'
```

Comment:
Codeception command produces an error report as a result of it's work therefore it needs to be parsed and converted into SDK's Violation Report. I assume codecept command has an option to change the report format to json so that it can be parsed and converted more easy.

Reply:
I added creation of the report after executing the command